app/redux/reducers/misc.js

```diff
@@ -150,10 +150,9 @@ export const allUsersMap = (state = initialState.allUsersMap, action) => {
         });
       } else {
         newState = update(state, {
-          [patient.userid]: { $set: _.omit(patient, ['permissions', 'team'])}
+          [patient.userid]: { $set: _.omit(patient, ['team'])}
```

Comment:
I wonder why `permissions` were originally omitted? I guess for `team` it makes sense, due to the below transformation into `others`. (Though I also don't follow why `permissions` are omitted there from the member.)